### DISCUSSION TITLE
Server connection issues

### DISCUSSION THREAD

**Discussion by ghost** [ansys/pyadditive#256] [Q&A] [answered]
Good mornig,

I have installed pyadditive-0.17 in developer mode using Ansys2023R1. When I run a built-in example I got the following error message:

"RuntimeError: Unable to connect to server 193.206.66.75:1055"

The host and the port are correct. I have tested them on Ansys Licensing Setting.

Any help would be appreciated. Thanks!

**Comment by RobPasMue** [comment]
Hi @maxcarrat90! Transferring this discussion to the PyAdditive repository. Pinging @pkrull-ansys for visibility as well

**Reply by ghost** [reply]
Thanks

**Comment by pkrull-ansys** [comment]
@maxcarrat90 The first release of the server was with Ansys 2024 R1. You'll need to upgrade to use pyadditive. Also, I recommend you use v0.15.0 of pyadditive with Ansys 2024 R1. If you decide to use v0.17.0, make sure to set the `product_version` parameter for the `Additive` initialization to `241`, https://additive.docs.pyansys.com/version/stable/api/ansys/additive/core/additive/Additive.html.

**Reply by ghost** [reply, marked as answer]
@pkrull-ansys Thank you for your reply. I have updated to Ansys 2024 R1 and modified the file server_connection/constants.py such that the DEFAULT_PRODUCT_VERSION = "241". 
Yet, when I try to run one of the file in the example folder I get the following error:
RuntimeError: Unable to connect to server 193.206.66.75:1055
The test in the Ansys Licensing Setting is passed so I assume both IP and port address are correct.

**Reply by pkrull-ansys** [reply]
@maxcarrat90 Thanks for your patience. I tried to duplicate your scenario by using v0.17.0 of pyadditive and Ansys 2024 R1. I was able to successfully run `examples/00_additive_single_bead.py` after modifying `DEFAULT_PRODUCT_VERSION`. 

What was at one time in the pyadditive docs but seems to have gotten removed is the fact that in order to run the solvers for the additive simulations, you'll need an Additive Suite license. Additive Suite is sold separately from other Ansys products. If you know you have an Additive Suite license, also referred to as the `am_module` increment, let me know and we can troubleshoot this further.

**Reply by ghost** [reply]
@pkrull-ansys Thank you very much for you hints. Now I am able to correctly run the `examples/00_additive_single_bead.py`. However, using `material = additive.material("316L")` , `machine.scan_speed = 1`, and `machine.laser_power = 195` I got zero values in all Melt Pool Dimensions, see plot below:
![MP_plot](https://github.com/ansys/pyadditive/assets/159887039/6a0abbbe-20f4-45d8-89ab-24973d9fe990)

**Reply by pkrull-ansys** [reply]
@maxcarrat90 Moving discussion to https://github.com/ansys/pyadditive/issues/261.
